components/identity-event/org.wso2.carbon.identity.event/src/test/java/org.wso2.carbon.identity.event/IdentityEventConfigBuilderTest.java

```diff
@@ -38,4 +39,10 @@ public void testGetInstance() throws IdentityEventException {
         IdentityEventConfigBuilder identityEventConfigBuilder =  IdentityEventConfigBuilder.getInstance();
         assertNotNull(identityEventConfigBuilder);
     }
+
+    @Test
+    public void testGetThreadPoolSize() throws IdentityEventException {
+
+        Assert.assertEquals("10",IdentityEventConfigBuilder.getInstance().getThreadPoolSize());
```

Comment:
Always use assert messages. So that if test fails you have a clue by looking at logs.